pkg/ebpf/ingress_node_firewall_events.go

```diff
@@ -28,20 +30,20 @@ func (infc *IngNodeFwController) ingressNodeFwEvents() {
 	// described in the eBPF C program.
 	rd, err := perf.NewReader(objs.IngressNodeFirewallEventsMap, os.Getpagesize())
 	if err != nil {
-		log.Printf("Failed creating perf event reader: %q", err)
-		return
+		return fmt.Errorf("Failed creating perf event reader: %q", err)
 	}
-	logFile, ok := os.LookupEnv("EVENT_LOGGING_FILE")
-	if !ok {
-		log.Printf("EVENT_LOGGING_FILE env variable must be set")
-		return
+
+	var eventsLogger *syslog.Writer
+	for i := 0; i < 30; i++ {
```

Comment:
pls use `wait.PollImmediate` to do this

Reply:
Done in https://github.com/openshift/ingress-node-firewall/compare/4242255efe2381100596f22f4d34a694fe2c9047..0213e5c988f44836d362f77d8fc12596ade13af7.